lib/mongoose_push/telemetry.ex

```diff
@@ -0,0 +1,31 @@
+defmodule MongoosePush.Telemetry do
+  @moduledoc """
+  Module responsible for handling telemetry events
+  """
+  alias MongoosePush.Telemetry.Handler
+
+  def handler do
+    attach_metrics()
+    {Handler, []}
+  end
+
+  defdelegate handle_event(tags, measurements, metadata, config), to: Handler, as: :handle_event
+
+  def attach_metrics do
+    _res =
+      :telemetry.attach_many(
+        "handler",
+        [
+          [:mongoose_push, :push],
```

Comment:
I feel that we should categorize the events more, maybe have separate handler modules for each service + core.
For this to happen, we need to change some event names, to always start off with one of:
* `[:mongoose_push, :apns, ....]`
* `[:mongoose_push, :fcm, ....]`
* `[:mongoose_push, :core, ....]`

Reply:
When we start handling sparrow events, we'll need to add more handler modules anyway, so that they are not too big and well scoped, so we may as well start here.